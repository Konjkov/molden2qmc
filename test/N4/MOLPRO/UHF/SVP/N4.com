*** Molpro input file for N4
memory,20,m
basis=def2-SVP
geomtyp=xyz
charge=1
geometry={
4                        ! number of atoms

 N,       0.5200,      0.5300,      0.5100
 N,      -0.5200,     -0.5200,      0.5200
 N,      -0.5200,      0.5200,     -0.5200
 N,       0.5200,     -0.5200,     -0.5200
}
uhf;save,mo

put,molden,N4_alpha.molden;orb,mo,set=1 ! alpha spins
put,molden,N4_beta.molden ;orb,mo,set=2 ! beta spins
