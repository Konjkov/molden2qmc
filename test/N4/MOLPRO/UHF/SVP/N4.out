
 Primary working directories    : /tmp/kbob
 Secondary working directories  : /tmp/kbob
 Wavefunction directory         : /home/kbob/wfu/
 Main file repository           : /tmp/kbob/

 SHA1      : 3480121182f1e6a142ecc57e7c34c53850cce301
 ARCHNAME  : Linux/x86_64
 FC        : /opt/intel/composer_xe_2013_sp1.1.106/bin/intel64/ifort
 FCVERSION : 14.0.1
 BLASLIB   : -Wl,-_start-group /opt/intel/mkl/lib/intel64/libmkl_intel_ilp64.a /opt/intel/mkl/lib/intel64/libmkl_sequential.a /opt/intel/mkl/lib/intel64/libmkl_core.a -Wl,-_end-group
 id        : vladimirkonjkov

 Nodes        nprocs
 kbob-G41MT-S2    1

 Using customized tuning parameters: mindgm=14; mindgv=24; mindgc=8; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  

 ***,Molpro input file for N4
 memory,20,m
 basis=def2-SVP
 geomtyp=xyz
 charge=1
 geometry={
 4                                                                               ! number of atoms
  N,       0.5200,      0.5300,      0.5100
  N,      -0.5200,     -0.5200,      0.5200
  N,      -0.5200,      0.5200,     -0.5200
  N,       0.5200,     -0.5200,     -0.5200
 }
 uhf;save,mo
 
 put,molden,N4_alpha.molden;orb,mo,set=1                                         ! alpha spins
 put,molden,N4_beta.molden ;orb,mo,set=2                                         ! beta spins

 Variables initialized (774), CPU time= 0.01 sec
 Commands  initialized (547), CPU time= 0.01 sec, 516 directives.
 Default parameters read. Elapsed time= 0.07 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                         Copyright, University College Cardiff Consultants Limited, 2008

                                    Version 2012.1 linked Nov 28 2013 13:06:21


 **********************************************************************************************************************************
 LABEL *   Molpro input file for N4                                                      
 Linux-3.13.0-32-generic/kbob-G41MT-S2(x86_64) 64 bit serial version                     DATE: 03-Jun-15          TIME: 18:57:17  
 **********************************************************************************************************************************

 SHA1:             3480121182f1e6a142ecc57e7c34c53850cce301
 **********************************************************************************************************************************

 Variable memory set to   20000000 words,  buffer space   230000 words

 SETTING BASIS          =    DEF2-SVP
 SETTING GEOMTYP        =    XYZ
 SETTING CHARGE         =         1.00000000                                  


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry N      S def2-SVP             selected for orbital group  1
 Library entry N      P def2-SVP             selected for orbital group  1
 Library entry N      D def2-SVP             selected for orbital group  1

1PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700


 Point group  C1  



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  N       7.00    0.982657588    1.001554850    0.963760327
   2  N       7.00   -0.982657588   -0.982657588    0.982657588
   3  N       7.00   -0.982657588    0.982657588   -0.982657588
   4  N       7.00    0.982657588   -0.982657588   -0.982657588

 Bond lengths in Bohr (Angstrom)

 1-2  2.792833659  1-3  2.766109823  1-4  2.779503859  2-3  2.779375377  2-4  2.779375377
     ( 1.477903921)     ( 1.463762276)     ( 1.470850094)     ( 1.470782105)     ( 1.470782105)

 3-4  2.779375377
     ( 1.470782105)

 Bond angles

  1-2-3   59.52518216   1-2-4   59.84274029   1-3-2   60.47940524   1-3-4   60.16082772

  1-4-2   60.31907745   1-4-3   59.68296140   2-1-3   59.99541260   2-1-4   59.83818225

  2-3-4   60.00000000   2-4-3   60.00000000   3-1-4   60.15621087   3-2-4   60.00000000

 NUCLEAR CHARGE:                   28
 NUMBER OF PRIMITIVE AOS:         100
 NUMBER OF SYMMETRY AOS:           96
 NUMBER OF CONTRACTIONS:           56   (  56A   )
 NUMBER OF CORE ORBITALS:           4   (   4A   )
 NUMBER OF VALENCE ORBITALS:       16   (  16A   )


 NUCLEAR REPULSION ENERGY  105.77794010


 Eigenvalues of metric

         1 0.734E-02 0.744E-02 0.754E-02 0.107E+00 0.130E+00 0.130E+00 0.131E+00 0.264E+00


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     6.291 MB (compressed) written to integral file ( 50.2%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:    1274406.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:    1274406      RECORD LENGTH: 524288

 Memory used in sort:       1.83 MW

 SORT1 READ     1605098. AND WROTE     1114218. INTEGRALS IN      4 RECORDS. CPU TIME:     0.03 SEC, REAL TIME:     0.06 SEC
 SORT2 READ     1114218. AND WROTE     1274406. INTEGRALS IN     24 RECORDS. CPU TIME:     0.08 SEC, REAL TIME:     0.11 SEC

 FILE SIZES:   FILE 1:    14.0 MBYTE,  FILE 4:    16.8 MBYTE,   TOTAL:     30.8 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       12.82       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         0.62      0.50
 REAL TIME  *         3.94 SEC
 DISK USED  *        31.11 MB      
 **********************************************************************************************************************************

1PROGRAM * UHF-SCF       Authors: W. Meyer, H.-J. Werner


 NUMBER OF ELECTRONS:      14+   13-    SPACE SYMMETRY=1    SPIN SYMMETRY: Doublet 
 CONVERGENCE THRESHOLDS:    1.00E-05 (Density)    1.00E-07 (Energy)
 MAX. NUMBER OF ITERATIONS:       60
 INTERPOLATION TYPE:            DIIS
 INTERPOLATION STEPS:              2 (START)      1 (STEP)
 LEVEL SHIFTS:                  0.20 (ALPHA)   0.20 (BETA) 



 Orbital guess generated from atomic densities. Full valence occupancy:   14

 Molecular orbital dump at record        2200.2

 Initial alpha occupation  14
 Initial beta  occupation  13

 ITERATION    DDIFF          GRAD             ENERGY        2-EL.EN.            DIPOLE MOMENTS         DIIS
    1      0.000D+00      0.000D+00      -216.58896569    331.716644   -0.00012    0.00364    0.00736    0
    2      0.172D-01      0.131D-01      -216.80571637    342.033838    0.00004    0.00474   -0.00410    1
    3      0.329D-02      0.175D-02      -216.81051132    342.016541   -0.00001    0.00547   -0.00179    2
    4      0.214D-02      0.908D-03      -216.81268977    342.128593    0.00003    0.00555   -0.00336    3
    5      0.104D-02      0.480D-03      -216.81359279    342.133782    0.00002    0.00603   -0.00265    4
    6      0.415D-03      0.203D-03      -216.81384576    342.122696    0.00012    0.00575   -0.00598    5
    7      0.116D-03      0.930D-04      -216.81390450    342.124146    0.00000    0.00645   -0.00139    6
    8      0.126D-03      0.699D-04      -216.81394613    342.126568    0.00010    0.00629   -0.00481    7
    9      0.136D-03      0.510D-04      -216.81397658    342.129084    0.00011    0.00627   -0.00430    7
   10      0.101D-03      0.359D-04      -216.81399267    342.128917    0.00014    0.00670   -0.00488    8
   11      0.726D-04      0.305D-04      -216.81400286    342.129165    0.00018    0.00630   -0.00514    9
   12      0.519D-04      0.291D-04      -216.81400991    342.129296    0.00020    0.00654   -0.00541    9
   13      0.132D-04      0.291D-04      -216.81401171    342.129275    0.00020    0.00653   -0.00547    9
   14      0.156D-03      0.292D-04      -216.81399109    342.128818    0.00016    0.00650   -0.00482    9
   15      0.532D-03      0.277D-04      -216.81393066    342.129736   -0.00006    0.00649   -0.00230    9
   16      0.616D-03      0.225D-04      -216.81388026    342.130371    0.00011    0.00651    0.00062    9
   17      0.214D-03      0.187D-04      -216.81386819    342.129945   -0.00016    0.00646    0.00150    9
   18      0.302D-03      0.166D-04      -216.81385621    342.130418    0.00019    0.00650    0.00296    9
   19      0.191D-03      0.144D-04      -216.81385222    342.130560   -0.00007    0.00653    0.00370    9
   20      0.458D-04      0.128D-04      -216.81385359    342.129376   -0.00016    0.00655    0.00376    9
   21      0.139D-04      0.121D-04      -216.81385412    342.130157   -0.00016    0.00656    0.00386    9
   22      0.351D-04      0.115D-04      -216.81385517    342.130066   -0.00021    0.00664    0.00392    9
   23      0.843D-04      0.112D-04      -216.81385919    342.129838   -0.00041    0.00667    0.00404    9
   24      0.196D-03      0.106D-04      -216.81386684    342.129882   -0.00082    0.00701    0.00425    9
   25      0.211D-03      0.906D-05      -216.81387357    342.130635   -0.00125    0.00715    0.00449    9
   26      0.191D-03      0.746D-05      -216.81387846    342.130272   -0.00167    0.00740    0.00446    9
   27      0.115D-03      0.572D-05      -216.81388043    342.130502   -0.00193    0.00740    0.00452    9
   28      0.105D-03      0.504D-05      -216.81388208    342.130188   -0.00209    0.00741    0.00442    9
   29      0.507D-04      0.445D-05      -216.81388274    342.130314   -0.00223    0.00746    0.00440    9
   30      0.174D-04      0.430D-05      -216.81388299    342.130264   -0.00223    0.00746    0.00441    9
   31      0.177D-04      0.424D-05      -216.81388324    342.130293   -0.00229    0.00749    0.00443    9
   32      0.224D-04      0.416D-05      -216.81388360    342.130369   -0.00231    0.00754    0.00447    9
   33      0.364D-04      0.406D-05      -216.81388420    342.130316   -0.00238    0.00757    0.00452    9
   34      0.641D-04      0.391D-05      -216.81388525    342.130399   -0.00245    0.00773    0.00455    9
   35      0.570D-04      0.370D-05      -216.81388611    342.130364   -0.00252    0.00779    0.00461    9
   36      0.791D-04      0.352D-05      -216.81388725    342.130414   -0.00265    0.00795    0.00457    9
   37      0.386D-04      0.337D-05      -216.81388777    342.130323   -0.00274    0.00803    0.00455    9
   38      0.291D-04      0.330D-05      -216.81388816    342.130356   -0.00280    0.00807    0.00454    9
   39      0.192D-04      0.329D-05      -216.81388842    342.130352   -0.00286    0.00811    0.00453    9
   40      0.306D-05      0.328D-05      -216.81388846    342.130354   -0.00288    0.00812    0.00452    9
   41      0.270D-05      0.329D-05      -216.81388850    342.130355   -0.00289    0.00813    0.00452    9
 
 Final alpha occupancy:  14
 Final beta  occupancy:  13

 !UHF STATE 1.1 Energy               -216.813888503788
 Nuclear energy                       105.77794010
 One-electron energy                 -493.65700608
 Two-electron energy                  171.06517748
 Virial quotient                       -1.00052807
 !UHF STATE 1.1 Dipole moment          -0.00289220     0.00812861     0.00451698
 Dipole moment /Debye                  -0.00735076     0.02065951     0.01148026

 Orbital energies for positive spin:

         1.1          2.1          3.1          4.1          5.1          6.1          7.1          8.1          9.1         10.1
    -16.125402   -16.124286   -16.120645   -16.120011    -2.050262    -1.423110    -1.413271    -1.401294    -1.027062    -0.935040

        11.1         12.1         13.1         14.1         15.1         16.1
     -0.925797    -0.922744    -0.916716    -0.895374    -0.183879    -0.163857

 Orbital energies for negative spin:

         1.1          2.1          3.1          4.1          5.1          6.1          7.1          8.1          9.1         10.1
    -16.115549   -16.114770   -16.112917   -16.112276    -2.019158    -1.377371    -1.348438    -1.325244    -1.017991    -0.910310

        11.1         12.1         13.1         14.1         15.1         16.1
     -0.895307    -0.892123    -0.884193    -0.456986    -0.141850    -0.107319

 HOMO    -13.1    -0.884193 =     -24.0601eV
 LUMO    -14.1    -0.456986 =     -12.4352eV
 LUMO-HOMO         0.427206 =      11.6249eV

 EXPECTATION VALUE OF S**2:             0.78550997

 Occupation number threshold for canonical orbitals:  1.999000 (occupied),  0.001000 (virtual)

 Number of pseudo-canonical orbitals per symmetry:   10
 Number of active natural orbitals per symmetry:      7

 Natural orbitals saved to record    2200.2 (Orbital set 3)

 Orbital spaces for UNO-CAS (Min. occupation: 0.020  Max. occupation: 1.980):

 Closed:       13
 Active:        1


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       12.82       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       4        0.48       700     1000      520     2200   
                                         GEOM     BASIS   MCVARS     UHF  

 PROGRAMS   *        TOTAL       UHF       INT
 CPU TIMES  *         1.59      0.97      0.50
 REAL TIME  *         5.36 SEC
 DISK USED  *        31.11 MB      
 **********************************************************************************************************************************

 Dump information in style MOLDEN to n4_alpha.molden                                                                 

 Molecular orbitals read from record     2200.2  Type=UHF/ALPHA (state 1.1)
 Occupation numbers read from record     2200.2  Type=UHF/ALPHA (state 1.1)
 Orbital energies read from record       2200.2  Type=UHF/ALPHA (state 1.1)
 Redundancy group numbers read from rec  2200.2  Type=UHF/ALPHA (state 1.1)
 
 DUMP ORBITAL  1.1 AS ORBITAL    1  occ=    1.0000  eig=  -16.1254  GROUP=   1
 DUMP ORBITAL  2.1 AS ORBITAL    2  occ=    1.0000  eig=  -16.1243  GROUP=   1
 DUMP ORBITAL  3.1 AS ORBITAL    3  occ=    1.0000  eig=  -16.1206  GROUP=   1
 DUMP ORBITAL  4.1 AS ORBITAL    4  occ=    1.0000  eig=  -16.1200  GROUP=   1
 DUMP ORBITAL  5.1 AS ORBITAL    5  occ=    1.0000  eig=   -2.0503  GROUP=   1
 DUMP ORBITAL  6.1 AS ORBITAL    6  occ=    1.0000  eig=   -1.4231  GROUP=   1
 DUMP ORBITAL  7.1 AS ORBITAL    7  occ=    1.0000  eig=   -1.4133  GROUP=   1
 DUMP ORBITAL  8.1 AS ORBITAL    8  occ=    1.0000  eig=   -1.4013  GROUP=   1
 DUMP ORBITAL  9.1 AS ORBITAL    9  occ=    1.0000  eig=   -1.0271  GROUP=   1
 DUMP ORBITAL 10.1 AS ORBITAL   10  occ=    1.0000  eig=   -0.9350  GROUP=   1
 DUMP ORBITAL 11.1 AS ORBITAL   11  occ=    1.0000  eig=   -0.9258  GROUP=   1
 DUMP ORBITAL 12.1 AS ORBITAL   12  occ=    1.0000  eig=   -0.9227  GROUP=   1
 DUMP ORBITAL 13.1 AS ORBITAL   13  occ=    1.0000  eig=   -0.9167  GROUP=   1
 DUMP ORBITAL 14.1 AS ORBITAL   14  occ=    1.0000  eig=   -0.8954  GROUP=   1

 Total charge:      14.000000

 **********************************************************************************************************************************

 Dump information in style MOLDEN to n4_beta.molden                                                                  

 Molecular orbitals read from record     2200.2  Type=UHF/BETA (state 1.1)
 Occupation numbers read from record     2200.2  Type=UHF/BETA (state 1.1)
 Orbital energies read from record       2200.2  Type=UHF/BETA (state 1.1)
 Redundancy group numbers read from rec  2200.2  Type=UHF/ALPHA (state 1.1)
 
 DUMP ORBITAL  1.1 AS ORBITAL    1  occ=    1.0000  eig=  -16.1155  GROUP=   1
 DUMP ORBITAL  2.1 AS ORBITAL    2  occ=    1.0000  eig=  -16.1148  GROUP=   1
 DUMP ORBITAL  3.1 AS ORBITAL    3  occ=    1.0000  eig=  -16.1129  GROUP=   1
 DUMP ORBITAL  4.1 AS ORBITAL    4  occ=    1.0000  eig=  -16.1123  GROUP=   1
 DUMP ORBITAL  5.1 AS ORBITAL    5  occ=    1.0000  eig=   -2.0192  GROUP=   1
 DUMP ORBITAL  6.1 AS ORBITAL    6  occ=    1.0000  eig=   -1.3774  GROUP=   1
 DUMP ORBITAL  7.1 AS ORBITAL    7  occ=    1.0000  eig=   -1.3484  GROUP=   1
 DUMP ORBITAL  8.1 AS ORBITAL    8  occ=    1.0000  eig=   -1.3252  GROUP=   1
 DUMP ORBITAL  9.1 AS ORBITAL    9  occ=    1.0000  eig=   -1.0180  GROUP=   1
 DUMP ORBITAL 10.1 AS ORBITAL   10  occ=    1.0000  eig=   -0.9103  GROUP=   1
 DUMP ORBITAL 11.1 AS ORBITAL   11  occ=    1.0000  eig=   -0.8953  GROUP=   1
 DUMP ORBITAL 12.1 AS ORBITAL   12  occ=    1.0000  eig=   -0.8921  GROUP=   1
 DUMP ORBITAL 13.1 AS ORBITAL   13  occ=    1.0000  eig=   -0.8842  GROUP=   1

 Total charge:      13.000000

 **********************************************************************************************************************************

        UHF-SCF 
   -216.81388850
 **********************************************************************************************************************************
 Variable memory released
